### ISSUE TITLE
Burn up

### ISSUE THREAD

**Issue by Shimwell** [ukaea/openmc_workshop#120]
OpenMC is implementing a burn up feature for fixed source simulations.

https://github.com/openmc-dev/openmc/pull/1628

Once this is merged in we should make an example that shows tbr as a function of time. Li6 burn up should mean tbr decreases as a function of time in regular blankets. Perhaps a layered solid blanket would be the best to highlight this effect

**Comment by Shimwell** [comment]
The burn up feature has now been merged into openmc development branch

**Comment by Shimwell** [comment]
to make this example there are some useful resources

The pin cell depletion example shows how to do burn up for fission sources
https://docs.openmc.org/en/latest/examples/pincell-depletion.html

**Comment by Shimwell** [comment]
The example made for the workshop should do two things

1) show the number of Li6 and L7 atoms as a function of time

2) show the TBR as a function of time

**Comment by Shimwell** [comment]
Andy D made a nice example that can obtain the number of atoms, thanks Andy
![Screenshot from 2020-09-30 16-51-26](https://user-images.githubusercontent.com/8583900/95445847-679bca80-0957-11eb-9bc1-42105ead8d90.png)
![Screenshot from 2020-09-30 16-51-18](https://user-images.githubusercontent.com/8583900/95445851-69658e00-0957-11eb-94ab-6819a02ca7d5.png)

**Comment by Shimwell** [comment]
Ah that last line in the above screen shot should be ```results.get_atoms("1", "Co60")```

**Comment by Shimwell** [comment]
Other useful links
API documentation
https://docs.openmc.org/en/stable/pythonapi/generated/openmc.deplete.Operator.html
